mongorover-0.1-1.rockspec

```diff
@@ -17,16 +17,17 @@ limitations under the License.
 --]]
 
 package = "mongorover"
- version = "0.0-1"
+ version = "0.1-1"
  source = {
-    url = "https://github.com/mongodbinc-interns/mongo-lua-driver"
+    url = "https://github.com/mongodbinc-interns/mongo-lua-driver",
```

Comment:
I forced pushed an update. It should be updated.

On Wed, Jul 15, 2015 at 4:53 PM, A. Jesse Jiryu Davis <
notifications@github.com> wrote:

> In mongorover-0.1-1.rockspec
> <https://github.com/10gen-labs/mongorover/pull/14#discussion_r34727804>:
>
> >   source = {
> > -    url = "https://github.com/mongodbinc-interns/mongo-lua-driver"
> > +    url = "https://github.com/mongodbinc-interns/mongo-lua-driver",
>
> 10gen-labs/mongorover now
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/10gen-labs/mongorover/pull/14/files#r34727804>.
>